modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/util/LanguageBatchUtil.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.gradle.plugins.workspace.internal.client.extension.util;
+
+import java.io.File;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Objects;
+
+import org.gradle.api.Project;
+
+/**
+ * @author Thiago Buarque
+ */
+public class LanguageBatchUtil {
+
+	public static boolean isSpecialLanguageProject(
```

Comment:
I may have missed more discussion around this, but shouldn't we just call this `isLanguageProject`?

Reply:
Ops, you are right. Missed that one